hack/setup/infra/manage.prometheus-helm.sh

```
@@ -0,0 +1,140 @@
+# Environment variables for custom Helm values:
+#   PROMETHEUS_EXTRA_ARGS - Additional helm install arguments
+#
+# Installs a minimal Prometheus stack suitable for CI:
+# - Prometheus server + ServiceMonitor CRD
+# - Grafana, AlertManager, and node-exporter disabled
+# - TLS enabled via cert-manager self-signed certificate (required by WVA)
+
+# INIT
+SCRIPT_DIR="$(cd "$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")" && pwd)"
+
+source "${SCRIPT_DIR}/../common.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./../common.sh was not specified as input (see shellcheck -x).

(SC1091)